apps/nextra/components/media/index.tsx

```diff
@@ -0,0 +1,12 @@
+export function YouTube({ title, id }: { title: string; id: string }) {
+  return (
+    <div>
```

Comment:
Can remove the div?

Reply:
Yeah, just copy pasted this from elsewhere